config/routes/sulu_admin.yaml

```diff
@@ -1,114 +1,102 @@
     prefix: /admin/api
 
 sulu_category_api:
-    type: rest
-    resource: "@SuluCategoryBundle/Resources/config/routing_api.yml"
+    resource: "@SuluCategoryBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
 
 sulu_snippet_api:
-    resource: "@SuluSnippetBundle/Resources/config/routing_api.yml"
-    type: rest
+    resource: "@SuluSnippetBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
 
 sulu_location:
-    resource: "@SuluLocationBundle/Resources/config/routing.yml"
+    resource: "@SuluLocationBundle/Resources/config/routing.yaml"
     prefix: /admin/location
 
 sulu_website:
-    resource: "@SuluWebsiteBundle/Resources/config/routing.yml"
+    resource: "@SuluWebsiteBundle/Resources/config/routing.yaml"
     prefix: /admin/website
 
 sulu_website_api:
-    resource: "@SuluWebsiteBundle/Resources/config/routing_api.yml"
-    type: rest
+    resource: "@SuluWebsiteBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
 
 sulu_core:
-    type: rest
-    resource: "@SuluCoreBundle/Resources/config/routing_api.yml"
+    resource: "@SuluCoreBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
 
 sulu_custom_urls_api:
-    type: rest
-    resource: "@SuluCustomUrlBundle/Resources/config/routing_api.yml"
+    resource: "@SuluCustomUrlBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
 
 sulu_route_api:
-    type: rest
-    resource: "@SuluRouteBundle/Resources/config/routing_api.yml"
+    resource: "@SuluRouteBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
 
 sulu_preview:
-    resource: "@SuluPreviewBundle/Resources/config/routing.yml"
+    resource: "@SuluPreviewBundle/Resources/config/routing.yaml"
     prefix: /admin/preview
 
 sulu_preview_api:
-    type: rest
-    resource: "@SuluPreviewBundle/Resources/config/routing_api.yml"
+    resource: "@SuluPreviewBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
 
 sulu_preview_public:
-    resource: "@SuluPreviewBundle/Resources/config/routing_public.yml"
+    resource: "@SuluPreviewBundle/Resources/config/routing_public.yaml"
     prefix: /admin/p
 
 sulu_search:
-    resource: "@SuluSearchBundle/Resources/config/routing.yml"
+    resource: "@SuluSearchBundle/Resources/config/routing.yaml"
     prefix: /admin/search
 
 sulu_activity_api:
-    resource: "@SuluActivityBundle/Resources/config/routing_api.yml"
-    type: rest
+    resource: "@SuluActivityBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
 
 sulu_trash_api:
-    resource: "@SuluTrashBundle/Resources/config/routing_api.yml"
-    type: rest
+    resource: "@SuluTrashBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
 
 sulu_reference_api:
-    resource: "@SuluReferenceBundle/Resources/config/routing_api.yml"
-    type: rest
+    resource: "@SuluReferenceBundle/Resources/config/routing_api.yaml"
     prefix: /admin/api
+
+#sulu_audience_targeting_api:
```

Comment:
Hmm should we add a link to the installation of the audience targeting bundle https://docs.sulu.io/en/2.6/bundles/audience_targeting.html ?

I'm not sure if it is obvious that this bundle has to be enabled first